### PR TITLE
Database: add primary key to device_graphs

### PR DESCRIPTION
**Pull request by rkojedzinszky**
Percona Xtradb refused to INSERT IGNORE into a table
without a primary key.

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: database/migrations/2020_03_22_1424_add_pkey_to_device_graphs.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        DB::table('device_graphs')->delete();
```
Comment (Jellyfrog) [inline database/migrations/2020_03_22_1424_add_pkey_to_device_graphs.php]: This can't be right?

Comment (rkojedzinszky) [reply database/migrations/2020_03_22_1424_add_pkey_to_device_graphs.php]: Actually, that table might contain duplicates, as no constraint prevented that. Thus, clear the table, then add the primary key. Also, poller modules will fill the table upon next run. Am I right?

Comment (murrant) [reply database/migrations/2020_03_22_1424_add_pkey_to_device_graphs.php]: @rkojedzinszky you can run through and delete duplicates first.

**Comment by murrant** [comment]
IMO, delete duplicates, add primary key id, add unique index on device_id and graph.

File: database/migrations/2020_03_22_1424_add_pkey_to_device_graphs.php (hunk 1 above)
Comment (rkojedzinszky) [reply database/migrations/2020_03_22_1424_add_pkey_to_device_graphs.php]: Unfortunately, on a table without a primary key, you cannot delete just duplicates. You cannot identify them :)

Comment (murrant) [reply database/migrations/2020_03_22_1424_add_pkey_to_device_graphs.php]: Add a new column with a primary key first then.

Comment (rkojedzinszky) [reply database/migrations/2020_03_22_1424_add_pkey_to_device_graphs.php]: Unfortunately, I am not a php developer, and perhaps, a bigger refactor would be needed here, which I cannot handle. The original problem involves a table without primary key and inserting into it with `insert ignore`. This breaks librenms on percona xtradb. Perhaps, the real solution is somewhere else.

**Pull request closed without merging**